contrib/drivers/filesharedrivers/drivers.go

```diff
@@ -43,23 +45,25 @@ type FileShareDriver interface {
 
 // Init
 func Init(resourceType string) FileShareDriver {
+	fmt.Println("resourceType ...", resourceType)
```

Comment:
Please remove the debug message, or use log.Infof.V(5) instead

Reply:
done!!